samples/GuestConfiguration/package-samples/README.md

```diff
@@ -12,42 +12,3 @@ However, this is not intended to be an Open Source project
 to manage the content used in Azure Policy as built-in content.
 
 ## Configurations
```

Comment:
should remove this header until you have some content to put under it

Reply:
Thanks. Fixed. I also removed one MOF. It won't hurt anything to have it in commit history. We just don't use it.